frontend/wallet/src/render/components/HelpButton.re

```diff
@@ -0,0 +1,154 @@
+type mode =
+  | HyperlinkBlue
+  | Gray
+  | Green
+  | Red
+  | OffWhite
+  | MidnightBlue
+  | HyperlinkBlue2
+  | HyperlinkBlue3
+  | LightBlue;
+
+module Styles = {
+  open Css;
+
+  let base =
+    merge([
+      Theme.Text.Body.regularSmall,
+      style([
+        display(`inlineFlex),
+        alignItems(`center),
+        justifyContent(`center),
+        background(white),
+        border(`px(0), `solid, white),
+        borderRadius(`rem(0.25)),
+        active([outlineStyle(`none)]),
+        focus([outlineStyle(`none)]),
+        disabled([pointerEvents(`none)]),
+      ]),
+    ]);
+
+  let buttonLink = style([textDecoration(`none), color(white)]);
```

Comment:
are these the same styles as above? Can you refactor these to use the same definitions?